docs/diagrams/SortFieldClassDiagram.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
+!include style.puml
+
+
+' Override style.puml
+show members
+hide empty members
+skinparam ClassAttributeIconSize 0
+skinparam Nodesep 100
+skinparam arrowThickness 1.1
+skinparam PackageBackgroundColor #FFFFFF
+
+
+' Class definitions
+Class ListPersonsCommand LOGIC_COLOR
+
+Class SortField MODEL_COLOR {
+    -comp : Comparator<Person>
+}
+
+Class "<<enumeration>>\nSortFieldType" as SortFieldType MODEL_COLOR {
+    NAME
+    DATE_OF_BIRTH
+    GENDER
+    NO_FIELD
+}
+
+package Model{
+    Class HiddenModel #FFFFFF
```

Comment:
It may be better to add `<<interface>>` in `Model` in the class diagram. You can also use `ModelManager` class instead of `Model` interface but Im not sure if it is better.

Reply:
For the class diagram I was following the Logic class diagram which showed `Model` as an empty white box without specifiying `<<interface>>`. In that diagram it looks like `Model` is to be viewed as a "black box" since its not the main focus of the diagram, which is why they don't draw it as a class.

My intention for the `SortFieldClassDiagram` is the same - I only want to show `Model` as a "black box" without going into its details 😄